.circleci/config.yml

```diff
@@ -46,6 +46,25 @@ jobs:
           name: Clone repo & compile project
           command: java -version && ./gradlew -v && ./gradlew --parallel testScoverageClasses
       - *save-gradle-cache
+  scala-style-hw:
+      <<: *machine-config
+      steps:
+        - checkout
+        - *open-jdk
+        - run:
+            name: Scala style check for HelloWorld
+            command: cd helloworld && ./gradlew --parallel scalaStyle && cd ..
+  compile-hw:
+      <<: *machine-config
+      steps:
+        - checkout
+        - *open-jdk
+        - *restore-gradle-cache
```

Comment:
I dont think you need to `*restore-gradle-cache` for Helloworld.

Reply:
fixed